recipes/clustalo/meta.yaml

```diff
@@ -1,25 +1,29 @@
-{% set version = "1.2.3" %}
+{% set version = "1.2.4" %}
 
 package:
   name: "clustalo"
   version: "{{ version }}"
 
 source:
-  fn: clustalo-{{ version }}-Ubuntu-x86_64 # [linux]
-  url: http://www.clustal.org/omega/clustalo-{{ version }}-Ubuntu-x86_64 # [linux]
-  md5: d6f953f498ea4fa1d08f57ea8cd731d0 # [linux]
-  fn: clustal-omega-{{ version }}-macosx # [osx]
-  url: http://www.clustal.org/omega/clustal-omega-{{ version }}-macosx # [osx]
-  md5: e8ec7292c258b8b1ef4f3780b3952fe8 # [osx]
+  url: http://www.clustal.org/omega/clustal-omega-{{ version }}.tar.gz # [not win]
+  sha256: 8683d2286d663a46412c12a0c789e755e7fd77088fb3bc0342bb71667f05a3ee # [not win]
+
+requirements:
+  build:
+    - gcc
```

Comment:
Can you please try to use llvm for osx as written in our documentation: https://bioconda.github.io/guidelines.html#c-c